price-feeder/oracle/provider/binance.go

```diff
@@ -239,7 +239,7 @@ func (p *BinanceProvider) getCandlePrices(key string) ([]types.CandlePrice, erro
 	return candleList, nil
 }
 
-func (p *BinanceProvider) messageReceived(messageType int, bz []byte) {
+func (p *BinanceProvider) messageReceived(_ int, bz []byte) {
```

Comment:
I think it would be better to have the name, rather than removing it, like `_messageType`. It serves documentation puroposes.

Reply:
In this case, hopefully going back to the `Provider` interface which it's implementing should work.

Otherwise, we could have added a `//nolint:` to some of these.